doc/TESTING.md

```diff
@@ -1,4 +1,4 @@
-# Testing
+# Local testing
 
 ### Unit & integration tests
```

Comment:
here, we go from `#` -> `###`. i,e h1 to h3. Consider adjusting these docs so they go incrementally in header size. 
In this case it would be `## Unit & integration tests`